table/table_test.cc

```diff
@@ -409,6 +409,10 @@ class TableConstructor: public Constructor {
 
   bool ConvertToInternalKey() { return convert_to_internal_key_; }
 
+  test::StringSink* GetSink() {
```

Comment:
Rename to `TEST_GetSink`?

Reply:
I don't know, they could all be considered test functions since this class is in a test file. But it's fine with me.